pymongo/encryption_options.py

```diff
@@ -236,10 +238,22 @@ def __init__(
         if not any("idleShutdownTimeoutSecs" in s for s in self._mongocryptd_spawn_args):
             self._mongocryptd_spawn_args.append("--idleShutdownTimeoutSecs=60")
         # Maps KMS provider name to a SSLContext.
-        self._kms_ssl_contexts = _parse_kms_tls_options(kms_tls_options)
+        self._kms_tls_options = kms_tls_options
+        self._sync_kms_ssl_contexts: dict[str, SSLContext] = None  # type:ignore[assignment]
+        self._async_kms_ssl_contexts: dict[str, SSLContext] = None  # type:ignore[assignment]
```

Comment:
Please remove `type:ignore[assignment]`. Those are bug factories and should only be used in exceptional circumstances.

Reply:
and fixed